recipe/meta.yaml

```diff
@@ -142,6 +144,7 @@ outputs:
         - idds-client >=0.11.0
         - idds-doma >=0.11.0
         - iminuit >=2
+        - importlib_resources
```

Comment:
We don't need this for python 3.11 because `importlib.resources` has the good API.